### DISCUSSION TITLE
Importer sur Mes Achats les données d'un tableur

### DISCUSSION THREAD

**Discussion by ghost** [betagouv/ma-cantine#1466] [Idées]
**Besoin d'une utilisatrice :**
Pouvoir importer sur Mes Achats des données renseignées dans un tableur, notamment dans le tableur fourni sur le site Ma Cantine.

**Intérêt :**
Basculer en cours d'année du tableur à Mes achats sans avoir à re-saisir toutes les factures déjà renseignées sur le tableur ni avoir un éclatement des données (première partie de l'année sur le tableur, seconde sur Mes Achats).

**Comment by valeriemerle** [comment]
Un mythe! mais oui à voir si ça serait possible; Le lycée agricole de Barbezieux a fait la saisie dans la version tableur et l'a transmise via le module mes achats (a glisser le fichier). Peux ton voir ce que cela donnerait?

**Reply by ghost** [reply]
À voir avec @alemangui, @hfroot et @JenniferStephan si c'est faisable.
@valeriemerle, comment ont-ils fait pour glisser-déposer le fichier dans Mes Achats? Je ne vois pas de tel module sur notre site.

**Comment by hfroot** [comment]
Sur staging il y a une première version de cette fonctionnalité (dès hier soir, annoncé sur le canal produit mattermost). Mais, il ne soutien pas le tableur qu'on fournit. À voir les améliorations à faire pour l'import avec les retours d'utilisateur.rice.s.

**Reply by ghost** [reply]
Parfait alors! Je viens de voir. Merci pour cette amélioration.
